core/trino-main/src/main/java/io/trino/execution/scheduler/faulttolerant/EventDrivenFaultTolerantQueryScheduler.java

```diff
@@ -109,6 +109,7 @@
 import it.unimi.dsi.fastutil.ints.IntOpenHashSet;
 import it.unimi.dsi.fastutil.ints.IntSet;
 import jakarta.annotation.Nullable;
+import jakarta.ws.rs.HEAD;
```

Comment:
Can you help me understand why this new `import` is necessary?

Reply:
:)

This is artifact of IntelliJ doing automatic imports. And conflict resolition with 
```
HEAD <<<<<<<<<<<
```

makers